File: app/src/main/java/com/quran/labs/androidquran/database/DatabaseUtils.java
package com.quran.labs.androidquran.database;

import android.database.Cursor;

public class DatabaseUtils {

  public static void closeCursor(Cursor cursor) {
    if (cursor != null) {
      try {
        cursor.close();
      } catch (Exception e) {
        // no op
      }
    }
  }
}

File: app/src/main/java/com/quran/labs/androidquran/database/DatabaseUtils.kt
package com.quran.labs.androidquran.database

import android.database.Cursor

object DatabaseUtils {
  @JvmStatic
  fun closeCursor(cursor: Cursor?) {
    try {
      cursor?.close()
    } catch (e: Exception) {
      // no op
    }
  }
}
